acoustid.py

```diff
@@ -348,6 +353,55 @@ def fingerprint_file(path, maxlength=MAX_AUDIO_LENGTH, force_fpcalc=False):
         return _fingerprint_file_fpcalc(path, maxlength)
 
 
+
+
+
+
+
+def compare_fingerprints(a, b) -> float:
+    """
+    compare two fingerprints locally
+    :param a: fingerprint of acoustid.fingerprint_file(filepath_a)
+    :param b: second fingerprint of acoustid.fingerprint_file(filepath_b)
+    :return:  similarity score [0,1]
+    """
+
+    if not have_chromaprint:
+        raise ModuleNotFoundError("function needs chromaprint")
+
+    # decompress fingerprints
+    a = chromaprint.decode_fingerprint(a)[0]
+    b = chromaprint.decode_fingerprint(b)[0]
+    a = list(map(int, a))
+    b = list(map(int, b))
```

Comment:
This could be a teensy bit more Pythonic:
```suggestion
    a = [int(x) for x in chromaprint.decode_fingerprint(a)[0]]
    b = [int(x) for x in chromaprint.decode_fingerprint(b)[0]]
```